src/components/Dissolution/AssociationDetails.vue

```diff
@@ -140,12 +141,18 @@ export default class AssociationDetails extends Mixins(CommonMixin, EnumMixin) {
   }
 
   /** Event handler for contact information changes. */
-  private async onContactInfoChange (event: BusinessContactIF): Promise<void> {
+  private async onContactInfoChange (event: ContactPointIF): Promise<void> {
+    // temporarily ignore data changes
+    this.setIgnoreChanges(true)
+
```

Comment:
Don't set the Have Changes flag here (whether user clicked OK or Cancel) because the filing data has not changed -- the contact info change was done in real time in Auth db.